src/tests/createHeadlessForm.test.js

```diff
@@ -994,6 +995,41 @@ describe('createHeadlessForm', () => {
       expect(fieldOptions).toEqual([]);
     });
 
+    it('support "integer" field type', () => {
```

Comment:
Hi @jguddas, forgive us the radio silence here. I'm picking this again today. I'm updating your branch with main, yet I noticed a test failing. I'll fix it tomorrow, and release a dev version to run against our internal CI/CD, and merge it :)

Once again, thanks for your contribution here!

Reply:
For reference: Internal [PR running the CI/CD](https://gitlab.com/remote-com/employ-starbase/dragon/-/merge_requests/33698)

Reply:
Internal CI is green, merging this 🎉